.github/workflows/conda.yml

```diff
@@ -30,7 +30,16 @@ jobs:
           submodules: true
           fetch-depth: 0 # history required so cmake can determine version
 
-      - uses: conda-incubator/setup-miniconda@v3
+      - name: Setup conda environment
+        uses: mamba-org/setup-micromamba@v2
+        with:
+          micromamba-version: 1.5.6-0
+          environment-name: build-env
+          create-args: >-
+            boa
```

Comment:
I don't think boa is supposed to be future proof. https://prefix-dev.github.io/rattler-build/latest/ is the future apparently :)